src/index.html

```diff
@@ -9,6 +9,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="#Home" class="header__logo" >
+          <img
+            class="logo"
+            src="images/log.png"
+            alt="Logo site"
+          >
+        </a>
+        <nav class="header__nav">
+          <ul class="nav_list">
+            <li class="menu_item">
+              <a href="#apple" class="menu_link is-active" >Apple</a>
```

Comment:
group css selectors as one block
```suggestion
        <nav class="nav">
          <ul class="nav_list">
            <li class="nav_item">
              <a href="#apple" class="nav_link is-active">Apple</a>
```